app/src/org/commcare/recovery/measures/ExecuteRecoveryMeasuresPresenter.java

```diff
@@ -298,7 +298,7 @@ private void showInstallMethodChooser() {
             }
         };
         d.setPositiveButton(StringUtils.getStringRobust(mActivity, R.string.recovery_measure_reinstall_online_method), listener);
-        d.setNegativeButton(StringUtils.getStringRobust(mActivity, R.string.recovery_measure_reinstall_offline_method), listener);
+        d.setNegativeButton(StringUtils.getStringRobust(mActivity, R.string.recovery_measure_reinstall_offline_method), listener, true);
```

Comment:
curious why was this changed ?

Reply:
We are now using an outlined button style for negative actions and the `Reinstall using CCZ` is not a negative action, so this is to change the style of the the negative button:
<img src="https://github.com/user-attachments/assets/7c01541b-44fb-48c7-8707-626bb115e261" width="150px" />

This is how it looked before:
<img src="https://github.com/user-attachments/assets/62480543-b29c-40eb-81f3-950f110546e5" width="150px"/>

Reply:
makes sense, thanks.